ui/page/send/layout.go

```diff
@@ -66,7 +68,21 @@ func (pg *Page) topNav(gtx layout.Context) layout.Dimensions {
 				return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
 					layout.Rigid(pg.infoButton.Layout),
 					layout.Rigid(func(gtx C) D {
-						return layout.Inset{Left: m}.Layout(gtx, pg.moreOption.Layout)
+						return layout.Inset{Left: m}.Layout(gtx, func(gtx C) D {
+							return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+								layout.Rigid(func(gtx C) D {
+									if pg.moreOptionIsOpen {
+										macro := op.Record(gtx.Ops)
+										layout.Inset{Top: values.MarginPadding25, Left: unit.Dp(-90)}.Layout(gtx, func(gtx C) D {
```

Comment:
move `unit.Dp(-90)` to dimensions.go file.
it should look something like this `values.MarginPaddingMinus90`